### ISSUE TITLE
Backpack Visibility 

### ISSUE THREAD

**Issue by Xraveler** [BeansGalaxy/Beans-Backpacks-3#14]
This mod is so cool, but it would be nice to have a toggleable model, because my player model is too small, but the bean is just floating there lol.

**Comment by BeansGalaxy** [comment]
Yeah it would be pretty easy to disable it but what do you mean it's too small?

**Comment by Xraveler** [comment]
Normally, when the player model is changed, all its parts are changed too. ex: armor, tools, shield etc. My custom player model is separate from this process though, so the backpack just floats there unchanged.

**Comment by Xraveler** [comment]
If I could make the toggle the model's invisibility, then it wouldn't look so glitchy.

**Comment by Xraveler** [comment]
Thank you BeansGalaxy.
